src/parser/druidsql.pegjs

```diff
@@ -637,6 +664,13 @@ Decorator =
   /'TRAILING'i
   /'DISTINCT'i
 
+JoinType =
+    'LEFT'i
+    /'RIGHT'i
+    /'INNER'i
+    /$('FULL'i _ 'OUTER'i)
+    /'FULL'i
+
 OrToken = 'OR'i
 AndToken = 'AND'i
 NotToken = 'NOT'i
```

Comment:
Could you please try to parse/stringily something like `Select notingham from table`

You might need a nice little `!IdentifierPart` on all of these. See: https://github.com/implydata/plywood/blob/master/src/expressions/plyql.pegjs#L1053

Reply:
`Select notingham from table` works fine but i will add a simple test anyway. The reason this isn't necessary is that in ply wood white space is always optional but in DQT is always enforced which will prevent the rule from matching on a string with out the need for !IdentifierPart.

Reply:
cool cool cool